### PR TITLE
TL translations - 1,277 words

### PR DESCRIPTION
**Pull request by josteinfrancois**
Hello everyone! I have a new pull request containing my translations contributions. Feel free to review my work.

### CODE REVIEW

File: book/09-git-and-other-scms/sections/import-hg.asc [hunk 1]
```diff
@@ -1,31 +1,31 @@
 ==== Mercurial
 
-(((Mercurial)))(((Importing, from Mercurial)))
-Since Mercurial and Git have fairly similar models for representing versions, and since Git is a bit more flexible, converting a repository from Mercurial to Git is fairly straightforward, using a tool called "hg-fast-export", which you'll need a copy of:
+(((Mercurial)))(((Pag-import, mula sa Mercurial)))
+Dahil ang Mercurial at Git ay patas na magkaparehong mga model para sa pag-representa ng mga bersyon, at dahil ang Git ay medyo mas umaangkop, ang pagpalit ng isang repositoryo mula sa Mercurial patungo sa Git ay patas na wagas, ang paggamit ng isang kasangkapan na tinatawag na "hg-fast-export", na kakailanganin mo ng isang kopya:
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/import-hg.asc]: model to modelo

File: book/09-git-and-other-scms/sections/import-hg.asc [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
 
 [source,console]
 ----
 $ hg clone <remote repo URL> /tmp/hg-repo
 ----
 
-The next step is to create an author mapping file.
-Mercurial is a bit more forgiving than Git for what it will put in the author field for changesets, so this is a good time to clean house.
-Generating this is a one-line command in a `bash` shell:
+Ang susunod na hakbang ay ang paggawa ng isang may-akda na mapping na file.
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/import-hg.asc]: mapping  na file to pagmamapa ng file

File: book/09-git-and-other-scms/sections/import-hg.asc [hunk 3]
```diff
@@ -1,31 +1,31 @@
...
+Ang Mercurial ay medyo mas mapagpatawad kaysa sa Git para sa kung ano ang ilalagay nito sa may-akda na field para sa mga hanay ng pagbabago, kaya ito ay isang magandang panahon upang maglinis ng bahay.
+Ang paglikha nito ay isang isang-linya ng utos sa isang `bash` na shell:
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/import-hg.asc]: isang isang-linya to iisang linya

File: book/09-git-and-other-scms/sections/import-hg.asc [hunk 4]
```diff
@@ -1,31 +1,31 @@
...
 
 [source,console]
 ----
 $ cd /tmp/hg-repo
 $ hg log | grep user: | sort | uniq | sed 's/user: *//' > ../authors
 ----
 
-This will take a few seconds, depending on how long your project's history is, and afterwards the `/tmp/authors` file will look something like this:
+Ito ay lilipas ng isang mga segundo, depende sa kung gaano kahaba ang kasaysayan ng iyong proyekto, at pagkatapos ang `/tmp/authors` na file ay magmumukhang katulad nito:
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/import-hg.asc]: isang to ilang

File: book/09-git-and-other-scms/sections/import-hg.asc [hunk 5]
```diff
@@ -50,7 +50,7 @@ bob <bob@company.com>=Bob Jones <bob@company.com>
 bob jones <bob <AT> company <DOT> com>=Bob Jones <bob@company.com>
 ----
 
-The next step is to create our new Git repository, and run the export script:
+Ang susunod na hakbang ay ang paggawa ng ating bagong Git na repositoryo, at patakbuhin ang export na iskrip:
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/import-hg.asc:53]: iskrip to script

File: book/09-git-and-other-scms/sections/import-hg.asc [hunk 6]
```diff
@@ -59,9 +59,9 @@ $ cd /tmp/converted
 $ /tmp/fast-export/hg-fast-export.sh -r /tmp/hg-repo -A /tmp/authors
 ----
 
-The `-r` flag tells hg-fast-export where to find the Mercurial repository we want to convert, and the `-A` flag tells it where to find the author-mapping file.
-The script parses Mercurial changesets and converts them into a script for Git's "fast-import" feature (which we'll discuss in detail a bit later on).
-This takes a bit (though it's _much_ faster than it would be over the network), and the output is fairly verbose:
+Ang `-r` na flag ay sasabihan ang hg-fast-export kung saan hahanapin ang Mercurial na repositoryo na gusto nating palitan, at ang `-A` na flag ay sasabihan ito kung saan hahanapin ang may-akda na pag-map na file.
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/import-hg.asc:62]: pag-map to pag-mapa

File: book/09-git-and-other-scms/sections/import-hg.asc [hunk 7]
```diff
@@ -59,9 +59,9 @@ $ cd /tmp/converted
 $ /tmp/fast-export/hg-fast-export.sh -r /tmp/hg-repo -A /tmp/authors
 ----
 
-The `-r` flag tells hg-fast-export where to find the Mercurial repository we want to convert, and the `-A` flag tells it where to find the author-mapping file.
-The script parses Mercurial changesets and converts them into a script for Git's "fast-import" feature (which we'll discuss in detail a bit later on).
-This takes a bit (though it's _much_ faster than it would be over the network), and the output is fairly verbose:
+Ang `-r` na flag ay sasabihan ang hg-fast-export kung saan hahanapin ang Mercurial na repositoryo na gusto nating palitan, at ang `-A` na flag ay sasabihan ito kung saan hahanapin ang may-akda na pag-map na file. 
+Ang iskrip ay magpa-parse ng Mercurial na hanay ng mga pagbabago at papalitan ang mga ito sa isang iskrip para sa "fast-import" na tampok ng Git (na ating tatalakayin nang detalyado sa kaunting saglit mamaya).
```
Comment (mind-blast) [inline book/09-git-and-other-scms/sections/import-hg.asc:63]: iskrip  to script

**Comment by josteinfrancois** [comment]
Thank you for your review @robin-ho . It has been updated.

**Pull request merged**